core/src/main/java/org/togglz/core/repository/FeatureState.java

```diff
@@ -119,50 +100,6 @@ public FeatureState disable() {
         return setEnabled(false);
     }
 
-    /**
-     * The list of users associated with the feature state.
-     *
-     * @return The user list, never <code>null</code>
-     *
-     * @deprecated This method will be removed soon. Use {@link #getParameter(String)} instead to read the corresponding
-     *             strategy parameter.
-     */
-    @Deprecated
-    public List<String> getUsers() {
```

Comment:
All three users methods seem quite handy in comparison to the get/setParameters method that only accepts a string.
Any good reason to drop them really, @bennetelli ?